src/Site.js

```diff
@@ -197,21 +197,17 @@ class Site {
     return false;
   }
 
-  readSiteConfig(baseUrl) {
-    return new Promise((resolve, reject) => {
+  async readSiteConfig(baseUrl) {
```

Comment:
Don't think there's an async operation going on here, can we remove it?

Reply:
The `serve` command in `index.js` expects the async nature unfortunately, didn't want to touch that here =X